packages/hop-node/src/db/TransfersDb.ts

```diff
@@ -241,15 +195,90 @@ class TransfersDb extends BaseDb {
     return normalizeDbItem(item)
   }
 
+  private readonly filterValueTransferId = (x: any) => {
+    return x?.value?.transferId
+  }
+
+  private async upsertTimestampedKeyItem (transfer: Transfer) {
+    const { transferId } = transfer
+    const logger = this.logger.create({ id: transferId })
+    const key = this.getTimestampedKey(transfer)
+    if (key) {
+      const exists = await this.subDbTimestamps.getById(key)
+      if (!exists) {
+        logger.debug(`storing db transfer timestamped key item. key: ${key}`)
+        await this.subDbTimestamps._update(key, { transferId })
+        logger.debug(`updated db transfer timestamped key item. key: ${key}`)
+      }
+    }
+  }
+
+  private async upsertRootHashKeyItem (transfer: Transfer) {
+    const { transferId } = transfer
+    const logger = this.logger.create({ id: transferId })
+    const key = this.getTransferRootHashKey(transfer)
+    if (key) {
+      const exists = await this.subDbRootHashes.getById(key)
+      if (!exists) {
+        logger.debug(`storing db transfer rootHash key item. key: ${key}`)
+        await this.subDbRootHashes._update(key, { transferId })
+        logger.debug(`updated db transfer rootHash key item. key: ${key}`)
+      }
+    }
+  }
+
+  private async upsertTransferItem (transfer: Transfer) {
+    const { transferId } = transfer
+    const logger = this.logger.create({ id: transferId })
+    await this._update(transferId, transfer)
+    const entry = await this.getById(transferId)
+    logger.debug(`updated db transfer item. ${JSON.stringify(entry)}`)
+    await this.upsertIncompleteItem(entry)
+  }
+
+  private async upsertIncompleteItem (transfer: Transfer) {
+    const { transferId } = transfer
+    const logger = this.logger.create({ id: transferId })
+    const isIncomplete = this.isItemIncomplete(transfer)
+    const exists = await this.subDbIncompletes.getById(transferId)
+    const shouldUpsert = isIncomplete && !exists
+    const shouldDelete = !isIncomplete && exists
+    logger.debug('storing db transfer incomplete key item')
```

Comment:
nitpick: it is possible that we do not store anything here. this log should be moved within the `if` statement or reworded.

Reply:
Good idea. The debug logs have moved and been updated 

Resolved with https://github.com/hop-protocol/hop/pull/351/commits/6fd88c22d80439b9ab8349efbb41c7b7f8b5b1ec